main.go

```diff
@@ -695,13 +675,12 @@ func main() {
 
 		err = runWatcher(fileChangeCommand, dirs, stopWatchServerOnChange)
 	} else {
-		Info.log("Watcher is not running.")
+		ControllerInfo.log("No APPSODY_RUN/TEST/DEBUG_ON_CHANGE action was not specified the file Watcher is not running.")
```

Comment:
This is an info message, lets make it very clear and obvious that the watcher is not running. Maybe just flip these around like:
"The file watcher is not running because no APPSODY_RUN/TEST/DEBUG_ON_CHANGE action was specified."